superset/assets/javascripts/explorev2/main.css

```diff
@@ -25,3 +25,8 @@
     margin-top: 0px;
     margin-right: 3px;
 }
+
+.control-panel-section {
```

Comment:
I don't know much about css classing best practices but I was wondering why introducing a new class as opposed to using existing ones, as in this selector: `.control-panel-section .panel` (of course assuming that we'd create a class for `control-panel-section` in the parent)